test/test_codescanner.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=wrong-import-order
+from pathlib import Path
+
+from objects.file import FileObject
+from test.unit.analysis.analysis_plugin_test_class import AnalysisPluginTest
+
+from ..code.codescanner import AnalysisPlugin
+
+TEST_DATA_DIR = Path(__file__).parent / 'data'
+
+
+class TestAnalysisPluginCryptoHints(AnalysisPluginTest):
+
+    PLUGIN_NAME = 'codescanner'
+    PLUGIN_CLASS = AnalysisPlugin
+
+    def test_process_object(self):
+        for test_file, expected_result in [
+            ('amd64.elf', 'Intel-64'),
+            ('arm.elf', 'Arm-32-le'),
+        ]:
```

Comment:
Do we have any test file that contains hidden executable data? That could be a nice addition to the testing here. :-)

Reply:
I added a file containing a raw bare metal binary between some random data. Is that what you had in mind?